babel-config.md

```diff
@@ -0,0 +1,17 @@
+## Babel configuration
+
+If your react project is not classic CRA (Create React App) make sure to add "@babel/plugin-transform-react-jsx-source to the project":
```

Comment:
Please, be more preceise and explain if there is babel.config.js in your project make sure that `@babel/plugin-transform-react-jsx-source` is installed and configured because there are other tools simmilar to CRA (like next.js, snowpack ...)